users/tests.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+from django.contrib.auth.models import User
+
+
+@pytest.mark.django_db
+def test_user_create():
+    User.objects.create_user('test', 'test@test.com', 'test')
+    count = User.objects.all().count()
+    print(count)
+    assert User.objects.count() == 1
```

Comment:
Before `User.objects.all().count()` was fetched but the final assert uses `User.objects.count()`.

Reply:
Fixed.